src/QsCompiler/CommandLineTool/Options.cs

```diff
@@ -99,6 +99,13 @@ public class CompilationOptions : Options
         internal TargetCapability TargetCapability =>
             TargetCapability.TryParse(this.TargetCapabilityName) ?? TargetCapabilityModule.Top;
 
+        [Option(
+            "error",
+            Required = false,
+            Default = null,
+            HelpText = "Comma-separated warning numbers that should be treated as errors.")]
+        public string? WarningAsErrorNumbers { get; set; }
```

Comment:
Why not `IEnumerable<int>`? I think that would avoid needing to manually parse the string?

Reply:
CompilationLoader constructs CompilationUnitManager with build properties represented as a string-string map. Other items are added via string representation to this collection. Therefore, I'm keeping the string representation until ProjectProperties are constructed.
If I use a collection of ins early on, I'd have to use a different way to construct ProjectProperties. Let's see if it works.